drivers/bus/isapnp/isapnp.c

```diff
@@ -175,7 +213,9 @@ IsaFdoCreateRequirements(
         }
     }
     if (ResourceCount == 0)
+    {
         return STATUS_SUCCESS;
+    }
```

Comment:
In other parts of your code you don't put braces for single-line ifs, so why do you add them just here? 🤔